odkx-src/conf.py

```diff
@@ -153,7 +153,7 @@
     'pointsize': '12pt',
 
     # Additional stuff for the LaTeX preamble.
-    'preamble': '''
+    'preamble': r'''
```

Comment:
What is this 'r' for?

Reply:
To not use the unicode escape. We end up compounding problems when trying to escape characters.

Reply:
It seems the '\u' was producing an error and the 'r' preceding the string turns off the eight-character Unicode escape